recipes/cad1/task2/data_preparation/build_scene_metadata.py

```diff
@@ -0,0 +1,168 @@
+"""Module to Generate the metadata for the scenes in the CAD-1 Task-2 challenge."""
+# pylint: disable=import-error
+
+import json
+import logging
+from typing import Any, Dict
+
+import hydra
+import numpy as np
+from omegaconf import DictConfig
+
+from clarity.utils.car_noise_simulator.carnoise_parameters_generator import (
+    CarNoiseParametersGenerator,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def set_seed(seed: int) -> None:
+    """Set the seed for the random number generator."""
+    np.random.seed(seed)
+
+
+def get_random_dict_item(input_dict: dict) -> Any:
+    """
+    Selects a random item from a dictionary.
+
+    Args:
+        input_dict (dict): A dictionary where keys are bird IDs and values are bird data.
+
+    Returns:
+        A random item from the input dictionary.
+    """
+    random_key = np.random.choice(list(input_dict.keys()), size=1, replace=False)[0]
+    return input_dict[random_key]
+
+
+def get_random_car_params(min_speed: int = 50, max_speed: int = 120) -> Dict:
+    """
+    Returns a dictionary with the parameters for a car noise.
+    These parameters are generated randomly, based on the car speed.
+    The parameters are to be used by the CarNoiseSignalGeneration class
+
+    Parameters:
+        - min_speed (int): The minimum speed that can be returned (default 50).
+        - max_speed (int): The maximum speed that can be returned (default 120).
+
+    Returns:
+        A dictionary containing the parameters needed by the CarNoiseSignalGeneration class.
+    """
+    speed = np.random.randint(min_speed, max_speed)
+    car_params = CarNoiseParametersGenerator().gen_parameters(speed)
+    return car_params
+
+
+def read_json(path_file, return_keys=False):
```

Comment:
Not sure about the typehint here but might be..
```suggestion
def read_json(path_file, return_keys=False) -> Union[Dict, List]:
```